tests/fixtures/mapping-resubmit.yml

```diff
@@ -0,0 +1,44 @@
+global:
+  default_inherits: default
+
+tools:
+  default:
+    cores: 2
+    mem: 12
+    env:
+      GALAXY_MEMORY_MB: "{mem * int(job.destination_params.get('SCALING_FACTOR', 1)) if job.destination_params else 1}"
```

Comment:
This means that GALAXY_MEMORY_MB is being set as a different value to the one being passed to slurm.  GALAXY_MEMORY_MB is usually calculated *from* the value that has been passed to slurm.  This mean that either (a) GALAXY_MEMORY_MB will be calculated from the slurm --mem and the environment variable will have no effect or (b) GALAXY_MEMORY_MB will be used in the tool wrapper and have a value possibly higher than has been allocated for the job.  I think it would probably be (a).

Reply:
This particular configuration is meant for the local runner, for which there doesn't appear to be any other way of specifying memory requirements?

Reply:
That's true, but do you ever want `GALAXY_MEMORY_MB` to have a different value from  `mem*1024`?  This all looks great, I am a fan of the tests with mock tools.